src/Particles/babylon.solidParticleSystem.ts

```diff
@@ -0,0 +1,427 @@
+module BABYLON {
+
+    export class SolidParticleSystem implements IDisposable {
+        // public members  
+        public particles = [];
+        public nbParticles:number = 0;
+        public billboard: boolean = false;
+        public counter: number = 0;
+        public name: string;
+        public mesh: Mesh;
+        
+        // private members
+        private _scene: Scene;
+        private _positions = [];
```

Comment:
oops
I fix it !

I coded it before I created the SolidParticle class

Le 14/10/2015 17:18, David Catuhe a écrit :
>
> In src/Particles/babylon.solidParticleSystem.ts 
> <https://github.com/BabylonJS/Babylon.js/pull/751#discussion_r42007408>:
>
> > @@ -0,0 +1,427 @@
> > +module BABYLON {
> > +
> > +    export class SolidParticleSystem implements IDisposable {
> > +        // public members
> > +        public particles = [];
> > +        public nbParticles:number = 0;
> > +        public billboard: boolean = false;
> > +        public counter: number = 0;
> > +        public name: string;
> > +        public mesh: Mesh;
> > +
> > +        // private members
> > +        private _scene: Scene;
> > +        private _positions = [];
>
> number[] ?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/BabylonJS/Babylon.js/pull/751/files#r42007408>.
>